wynnset/tapestry-wp
# Fix videos getting clipped in some cases

Fixes #469 

Testing steps:

- Create a video node and choose a video that is wider than it is tall
- Set the node appearance to fullscreen
- Resize your browser so it is much wider than it is tall
- The video should fit the browser and be centered
